File: 30-days-of-code/Day05/loops.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    loops.py                                           :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: alaamimi <alaamimi@student.42.fr>          +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2021/03/06 16:54:41 by alaamimi          #+#    #+#              #
#    Updated: 2021/03/06 17:28:41 by alaamimi         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

#!/bin/bash 

import math
import os
import random
import re
import sys

if __name__ == '__main__':
    n = int(input())
    for i in range(1, 11):
        product = n * i
        print('{} x {} = {}'.format(n, i, product))

File: 30-days-of-code/Day05/Loops.java
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   Loops.java                                         :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: alaamimi <alaamimi@student.42.fr>          +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2021/02/25 19:31:03 by alaamimi          #+#    #+#             */
/*   Updated: 2021/02/25 19:31:06 by alaamimi         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

import java.util.Scanner;

public class Solution
{
	private static final Scanner scanner = new Scanner(System.in);
	public static void main(String[] args)
	{
		int n = scanner.nextInt();
		scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");   
		int i;

		i = 1;
		while (i <= 10)
		{
			System.out.print(n);
			System.out.print(" x ");
			System.out.print(i);
			System.out.print(" = ");
			System.out.println(n * i);
			i++;
		}
		scanner.close();
	}
}
